doc/source/tech_note/Photosynthetic_Capacity/CLM50_Tech_Note_Photosynthetic_Capacity.rst

```diff
@@ -53,65 +53,65 @@ The structural nitrogen, :math:`N_{\text{str}}`, is calculated as the multiplica
+ +----------------------------------+--------------------------+
+ | BET Tropical                     |          0.01900         |
+ +----------------------------------+--------------------------+
+ | BET temperate                    |          0.01900         |
+ +----------------------------------+--------------------------+
+ | BDT tropical                     |          0.03080         |
+ +----------------------------------+--------------------------+
+ | BDT temperate                    |          0.03080         |
+ +----------------------------------+--------------------------+
+ | BDT boreal                       |          0.03080         |
+ +----------------------------------+--------------------------+
+ | BES temperate                    |          0.01798         |
+ +----------------------------------+--------------------------+
+ | BDS temperate                    |          0.03072         |
+ +----------------------------------+--------------------------+
+ | BDS boreal                       |          0.02800         |
+ +----------------------------------+--------------------------+
+ | C\ :sub:`3` arctic grass         |          0.02100         |
+ +----------------------------------+--------------------------+
+ | C\ :sub:`3` grass                |          0.04024         |
+ +----------------------------------+--------------------------+
+ | C\ :sub:`4` grass                |          0.03846         |
+ +----------------------------------+--------------------------+
+ | Temperate Corn                   |          0.05000         |
+ +----------------------------------+--------------------------+
+ | Spring Wheat                     |          0.03500         |
+ +----------------------------------+--------------------------+
+ | Temperate Soybean                |          0.03500         |
+ +----------------------------------+--------------------------+
+ | Cotton                           |          0.03500         |
+ +----------------------------------+--------------------------+
+ | Rice                             |          0.03500         |
+ +----------------------------------+--------------------------+
+ | Sugarcane                        |          0.05000         |
+ +----------------------------------+--------------------------+
+ | Tropical Corn                    |          0.05000         |
+ +----------------------------------+--------------------------+
+ | Tropical Soybean                 |          0.03500         |
+ +----------------------------------+--------------------------+
+ | Miscanthus                       |          0.03500         |
+ +----------------------------------+--------------------------+
+ | Switchgrass                      |          0.03500         |
+ +----------------------------------+--------------------------+
+
+Notes: :math:`SLA_{\text{0}}` is the specific leaf area at the canopy top (m :sup:`2` leaf/g biomass)

```

Comment:
- [ ] I actually refer to $N_{cb}$ in #2221. Is there a reason you got rid of it here?

Reply:
It's not actually a parameter in the parameter file.